tests/periph_uart/main.c

```diff
@@ -28,6 +28,7 @@
 #include "msg.h"
 #include "ringbuffer.h"
 #include "periph/uart.h"
+#include "uart_stdio.h"
```

Comment:
see ef0aef3164fe267bbdb17a365518bf7e82a9b19c
This makes sure UART_STDIO_DEV is defined, so the init command fails when trying to use the shell UART as a parameter.
Maybe there's a better way to do this?